src/dmd/func.d

```diff
@@ -1419,6 +1420,14 @@ extern (C++) class FuncDeclaration : Declaration
         }
         else if (isSafe())
             return true;
+        else if (global.params.showSystem &&
+                 type.toTypeFunction().trust == TRUST.default_ &&
+                 !(flags & FUNCFLAG.warnedSystem))
+        {
+            flags |= FUNCFLAG.warnedSystem;
+            message(loc, "%s is un-@safe but not marked as @system", ident.toChars);
```

Comment:
```suggestion
            message(loc, "function `%s` default safety will change from `@system` to `@safe`", ident.toPrettyChars());
```

It doesn't have to be exactly this wording, but "un-@safe" is not used in the compiler and might not be clear to the user, while `@system` and `@safe` are. The rest are just cosmetic fixes.
Other possible wording is just to say "... defaults to `@system`".

Reply:
Removed the "un-`@safe`" and adapted the message to hint at the missing `@system`.